.changes/v3.8.1/949-bug-fixes.md

```diff
@@ -0,0 +1 @@
+* Fix issue #944 shared catalog datasource not working [GH-949]
```

Comment:
This sounds really harsh :) (as if it was not working at all). Would you add more context like "when" it was not working?